scenario/README.md

```diff
@@ -0,0 +1,108 @@
+
+### Dam break scenario
+
+This scenario simulate the motion of a fluid block under the effect of gravity.
+The simulators available are [SPlisHSPlasH](https://github.com/inductiva/inductiva/blob/main/inductiva/simulators/splishsplash.py) and [DualSPHysics](https://github.com/inductiva/inductiva/blob/main/inductiva/simulators/dualsphysics.py), which use the [Smoothed Particle Hydrodynamics](https://en.wikipedia.org/wiki/Smoothed-particle_hydrodynamics) formulation to model the fluid.
+
+One can initialize the scenario with different fluids (Water, Honey, Olive oil) and with fluid blocks of different dimensions and initial positions. Then, users can further configure the simulation time and resolution.
+
+##### Example
+
+We have prepared a Python script to directly run this scenario. Get inside the scenario folder and run the following command:
+
+```bash
+python3 dam_break_run.py
+```
+
+Or, prepare your own and start changing the parameters. The script should be inside the scenario folder. Copy the code snippet and get started:
+
+```python
+import dam_break
+import post_processing
+from utils import fluid_types
+
+# Initialize the DamBreak scenario
+dam_break = dam_break.DamBreak(fluid=fluid_types.WATER)
+
+# Launch the simulation
+task = dam_break.simulate(
+    simulation_time=4,
+    resolution="medium")
+
+# Wait for the simulation to finish and download the results
+task.wait()
+output_dir = task.download_outputs()
+
+# Post-process the results and render a video of the simulation
+simulation_output = post_processing.SPHSimulationOutput(output_dir)
+simulation_output.render(fps=60)
+```
+
+<p align="center">
+  <img src="assets/dam_break.gif" alt="Centered Image" width="400" height="300">
+
+
+### Fluid Block
+
+This scenario is an extension of the DamBreak scenario, where the fluid can have an initial velocity and the simulation can be configured with more parameters. 
+
+To initialize the scenario users can set the density and kinematic viscosity of the fluid, the dimensions of the fluid block, the initial position and the initial velocity. 
+
+The simulation can be configured with the following parameters: the `particle_radius` (float), the `simulation_time` (float), the `adaptive_time_step` (bool), the `time_step` (float) and the `output_export_rate` (int).
+
+Again, the scenario can be run with both `SplishSplash` and `DualSPHysics` simulators via Inductiva API.
+
+##### Example
+
+We have prepared a Python script to directly run this scenario. Get inside the scenario folder and run the following command:
+
+```bash
+python3 fluid_block_example.py
+```
+
+Or, prepare your own and start changing the parameters. The script should be inside the scenario folder. Copy the code snippet and get started:
+
+```python
+import fluid_block
+import post_processing
+
+# Initialize the FluidBlock scenario
+fluidblock = fluid_block.FluidBlock(
```

Comment:
This needs to be explained in the docs.
We created this class to store the parameters of part of the simulation.
My point is that we should be writing not just on how we use the scenario (which is totally toy) but more importantly on how we create a scenario to "scale" experimentation.

So, with this repo we can show how we create on scenario, and then how we extend it to build another.

Reply:
I don't want to repeat things that are already explained in our Inductiva Documents. However, for story telling purposes I will add here a brief description.

However, note that the dambreak is not an extension, it actually is a restriction at the moment. Users will need to create the FluidBlock first.